go.mod

```diff
@@ -20,9 +20,9 @@ require (
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/lib/pq v1.10.9
 	github.com/lightninglabs/aperture v0.3.2-beta
-	github.com/lightninglabs/lndclient v0.18.0-0
+	github.com/lightninglabs/lndclient v0.17.4-4
```

Comment:
`lndclient` was downgraded from v0.18.0-0 to v0.17.4-4. Can  `v0.18.0-0` be used?

Also, `lndclient v0.18.0-0` uses lnd `v0.18.0-beta.rc3`. Does it make sense to bump lndclient first to `lnd `v0.18.0-beta` first?

Reply:
I just pushed a new version `v0.18.0-1` that references the final `lnd v0.18.0-beta` tag. So you can use that.